nix/iogx-config.nix

```diff
@@ -0,0 +1,15 @@
+{
+  debug = true;
```

Comment:
Should this be `false`?

Reply:
no difference, it's disappearing in the next push to this PR.